localnet/poktrolld/config/appgate_server_config.yaml

```diff
@@ -0,0 +1,8 @@
+# Whether the server should sign all incoming requests with its own ring (for applications)
+self_signing: true
+# The name of the key that will be used to sign relays
+signing_key: app1
+# The host and port that the appgate server will listen on
+listening_endpoint: http://localhost:42069
```

Comment:
DISCUSSION (not asking to change it).

DO you think `appgate_listening_endpoint` would be too verbose?

Reply:
I believe using `listening_endpoint` is appropriate as long as the context remains clear, such as within the AppGate context. If there's a risk of losing context due to file name or lack of comments at the beginning of the file, then we can consider 'appgate_listening_endpoint' for better clarity.